src/components/silhouette-pane.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { clsx } from "clsx";
+
+import { PaneTitle } from "./pane-title";
+import { AppContext } from "../hooks/use-app-context";
+import { ISilhouetteData, ISilhouetteOrganData, silhouetteData, silhouetteOrganData }
+  from "../assets/app-data/silhouette-data";
+
+import "./silhouette-pane.scss";
+
+// Make sure you update $transition-duration in silhouette-pane.scss if you change this.
```

Comment:
This is clear and commented in both places so I think it makes sense for maintainability.